apps/nextjs-website/react-components/types/MediaResources/MediaResources.types.ts

```diff
@@ -0,0 +1,32 @@
+import { SectionProps } from '../common/Common.types';
+
+export interface MediaResourcesItem {
+  title: string;
+  thumbnail: {
+    data: {
+      attributes: {
+        url: string;
+      };
+    };
+  };
+  resource: {
+    data: {
+      attributes: {
+        url: string;
+      };
+    };
+  };
+  icon: {
+    data: {
+      attributes: {
+        url: string;
+      };
+    };
+  };
+  label: string;
+}
```

Comment:
```suggestion
export interface MediaResourcesItem {
  title: string;
  thumbnailURL: string;
  resourceURL: string;
  label: string;
}
```
Simplify.

We'll parse data inside a `MakeMediaResourcesProps` function as always